cpu/cortexm_common/Makefile.features

```diff
@@ -0,0 +1 @@
+FEATURES_PROVIDED += cpp
```

Comment:
Then this PR is missing the removal of all the `cpp` entries from all the cortexm boards, but this is part of the `WIP` right?

Reply:
correct!

Reply:
done